GVFS/GVFS/RepairJobs/GitConfigRepairJob.cs

```diff
@@ -16,7 +16,7 @@ public GitConfigRepairJob(ITracer tracer, TextWriter output, GVFSEnlistment enli
 
         public override string Name
         {
-            get { return ".git\\config"; }
+            get { return $".git{Path.DirectorySeparatorChar}config"; }

```

Comment:
These changes don't seem like they are related to this PR.  I understand why you are making them but can they be in a separate PR.  Also they should be using the `GVFSConstants.DotGit` properties I would think.

Reply:
I've pulled these out and tidied them up further in PR #1448 -- thanks for the advice!